### PR TITLE
SDK-649: Update Readme to reflect the new endpoints from the breaking changes

### PR DESCRIPTION
**Pull request by sidibos**
I have added getReceiptId() and getTimestamp() to this PR Ref: [SDK-561.](https://lampkicking.atlassian.net/browse/SDK-561)
Could you review it again please!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,48 +124,51 @@ $givenNames         = $profile->getGivenNames()->getValue();
...
+
+$ageUnderXX         = $profile->findAgeUnderVerification($xx); // AgeVerification Object (please see API Coverage for details), Where $xx is the age value, e.g 18
 
-$verifiedAge        = $profile->getVerifiedAge()->getValue();
 ```
 
 ## Handling Users
 
-When you retrieve the user profile, you receive a user ID generated by Yoti exclusively for your application.
+When you retrieve the user profile, you receive a rememberMe ID generated by Yoti exclusively for your application.
```
Comment (echarrod) [inline README.md]: I think this should either be `Remember Me Id` or `RememberMeId` (2 spaces or no spaces). It looks like it's called `rememberMeId` in the dev docs, so I would lean towards that.

File: README.md [hunk 2]
```diff
@@ -125,48 +124,51 @@ $givenNames         = $profile->getGivenNames()->getValue();
...
 
 ```php
 <?php
-$activityDetails = $client->getActivityDetails($token);
-$profile = $activityDetails->getProfile();
-
-if ($client->getOutcome() == \Yoti\YotiClient::OUTCOME_SUCCESS) {
-    $user = yourUserSearchFunction($activityDetails->getUserId());
+try {
+    $activityDetails = $client->getActivityDetails($token);
```
Comment (echarrod) [inline README.md]: Could we change this to `oneTimeUseToken` as per https://lampkicking.atlassian.net/browse/SDK-209? (I know it still has to be `token` in the get request). Also we mention `token` other times in the README, please can you change those too?

File: README.md (hunk 1 above)
Comment (sidibos) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (sidibos) [reply README.md]: Not sure if using $oneTimeUseToken will be clearer as than the comments I have added [here](https://github.com/getyoti/yoti-php-sdk/tree/SDK-649#profile-retrieval)

**Pull request merged**